src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -156,6 +156,17 @@ object DockerPlugin extends AutoPlugin {
     dockerBuildOptions := Seq("--force-rm") ++ dockerAliases.value.flatMap { alias =>
       Seq("-t", alias.toString)
     } ++ { if (dockerBuildInit.value) List("--init") else Nil },
+    dockerBuildEnvVars := Map.empty,
+    dockerBuildkitEnabled := {
+      dockerBuildEnvVars.value
+        .get("DOCKER_BUILDKIT")
+        .orElse(sys.env.get("DOCKER_BUILDKIT"))
+        .map(_ == "1")
```

Comment:
Docker uses [strconv.ParseBool](https://pkg.go.dev/strconv#ParseBool) which accepts some other values, but we'll assume the value is either `1` or `0`.